susemanager-utils/dracut-mgrbootstrap/mgrbootstrap/mgrbootstrap.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+if ! declare -f Echo > /dev/null ; then
+  Echo() {
+    echo "$@"
+  }
+fi
+
+if ! [ -s /etc/resolv.conf ] ; then
+    Echo "No network, skipping saltboot..."
+    exit 0
+fi
+
+IFCONFIG="$(compgen -G '/tmp/leaseinfo.*.dhcp.ipv*')"
+if [ -f "$IFCONFIG" ]; then
+    . "$IFCONFIG"
+else
+    Echo "No network details available, skipping saltboot";
+    exit 0
+fi
```

Comment:
Later you are calling `ip` directly, so you don't need this bits to source ip configuration.

Reply:
ok